ansible/roles/test/tasks/bgp_speaker.yml

```diff
@@ -117,6 +134,47 @@
   assert: {that: "'{{ bgp_neighbors[item]['accepted prefixes'] }}' == '1'"}
   with_items: "['{{speaker_ips[0].split('/')[0]}}', '{{speaker_ips[1].split('/')[0]}}', '{{vlan_ips[0].split('/')[0]}}'] "  
 
+# Send packets to verify that accepted prefixes are correctly applied in HW.
+# PTF FIB test is used to send the packets.
+
+- name: Generate route-port map information
+  template: src=roles/test/templates/bgp_speaker_route.j2
+            dest=/tmp/bgp_speaker_route.txt
+  connection: local
+
+- name: Copy the bgp_speaker_route to ptf container
+  copy: src=/tmp/bgp_speaker_route.txt dest=/root
+  delegate_to: "{{ptf_host}}"
+
+- name: Copy the test to ptf container
+  copy: src=roles/test/files/ptftests dest=/root
+  delegate_to: "{{ ptf_host }}"
+
+- set_fact: ipv4=true
+  when: addr_family == 'ipv4'
+
+- set_fact: ipv6=true
+  when: addr_family == 'ipv6'
+
+- name: "Start PTF runner"
+  include: ptf_runner.yml
+  vars:
+    ptf_test_name: FIB test
+    ptf_test_dir: ptftests
+    ptf_test_path: fib_test.FibTest
+    ptf_platform: remote
+    ptf_test_params:
+      - testbed_type='{{testbed_type}}'
+      - router_mac='{{ansible_Ethernet0['macaddress']}}'
+      - fib_info='/root/bgp_speaker_route.txt'
+      - ipv4={{ ipv4|default(false) }}
```

Comment:
default true for ipv4?

Reply:
Yes and default value is set to IPv4 in line 80.